app/models/model.py

```diff
@@ -271,76 +202,48 @@ class FaceAbridged(BaseModel):
 
     geometry: Face3D
 
-    properties: FaceProperties
-
-    face_type: Union[Wall, RoofCeiling, Floor, AirWall,  Shading]
-
     boundary_condition: Union[Ground, Outdoors, Adiabatic, Surface]
```

Comment:
The only 2 boundary conditions that are legal for an `Aperture` are `Outdoors` and `Surface`.  So you should remove `Ground` and `Adiabatic` from here.